source/Calamari.Shared/Integration/FileSystem/CalamariPhysicalFileSystem.cs

```diff
@@ -457,7 +457,7 @@ public void OverwriteAndDelete(string originalFile, string temporaryReplacement)
                 File.Copy(temporaryReplacement, originalFile, true);
             else
             {
-                System.IO.File.Replace(temporaryReplacement, originalFile, backup);
+                System.IO.File.Replace(temporaryReplacement, originalFile, backup, true);
```

Comment:
Hard to tell from the diff in github, but this is the `ignoreMetadataErrors` param, correct?

> ignoreMetadataErrors
> Boolean
> true to ignore merge errors (such as attributes and access control lists (ACLs)) from the replaced file to the replacement file; otherwise, false.

If so, this seems... dangerous. We could end up with files with incorrect ACLs, either breaking Octopus or ending up with files readable by people who arent meant to.

Reply:
Both files have the same acl so wouldn’t be an issue on the merge. Other option would be to set true for unc paths only.